AmplifyPlugins/Core/AWSPluginsCoreTests/Sync/PaginatedListTests.swift

```diff
@@ -94,4 +95,46 @@ class PaginatedListTests: XCTestCase {
         }
     }
 
+    func testDecodePaginatedListOptimastically() {
+        let syncQueryJSON = """
```

Comment:
we can declare this JSON along with other JSONs at the top and name it something like `syncQueryJSONWithMissingFields`

Reply:
we can also have a test with null object in `items` array: 
```
"items" : [
   null,
   { post }
]
```

Reply:
will add `null` as a case

Reply:
thanks for the updates, we should move the declaration of this new JSON string at the top of the file like at line 42

Reply:
This is not shared across test cases. Including test data inside the test case is easier to read.